spec/services/cerc_forecast_service_spec.rb

```diff
@@ -1,6 +1,6 @@
 RSpec.describe CercForecastService do
   describe "::latest_forecasts_for(zone)" do
-    let(:zone) { FactoryBot.build(:zone, name: "Barnet") }
+    let(:zone) { create(:zone, cerc_id: 29) }
```

Comment:
if we're being consistent here, should we leave it as `FactoryBot.build`?

Reply:
Good spot!